docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -111,13 +108,13 @@ app:
 
 == Profiles
 
-As you can see in the previous snippet, you can use xref:config-reference.adoc#profiles[profiles] in YAML. The profile
-key requires double quotes: `"%test"`. This is because YAML does not support keys starting with `%`.
+As you can see in the previous snippet, you can use link:{url-quarkusio-guides}/config-reference.adoc#profiles[profiles] in YAML.
+
+In YAML, keys that begin with `%` are not allowed. However, profile keys often start with this symbol. To resolve this, enclose the profile keys in double quotes, as demonstrated by the example, `"%test"`.
 
-Everything under the `"%test"` key is only enabled when the `test` profile is active. For example, in the previous
-snippet it disables OIDC (`quarkus.oidc.enabled: false`), whereas without the `test` profile, it would be enabled.
+All configurations under the `"%test"` key activate only when the `test` profile is enabled. For instance, the previous snippet shows that OpenID Connect (OIDC) (`quarkus.oidc.enabled: false`) is disabled when the `test` profile is active. Without the `test` profile, OIDC is enabled by default.
 
-As for the Java Properties format, you can define your own profile:
+For the Java Properties format, you can define your own profile:
```

Comment:
Should the P in `Properties` be uppercase? Downstream and other places it's lowercase.

 ```suggestion
For the Java properties format, you can define your own profile:
```

Reply:
Updated. Thanks.